bin/citrea/src/rollup/mod.rs

```diff
@@ -43,9 +44,21 @@ pub trait CitreaRollupBlueprint: RollupBlueprint {
         let mut storage_manager = self.create_storage_manager(&rollup_config)?;
         let prover_storage = storage_manager.create_finalized_storage()?;
 
+        let (soft_confirmation_tx, soft_confirmation_rx) = broadcast::channel(10);
+        // If subscriptions disabled, pass None
+        let soft_confirmation_rx = if !rollup_config.rpc.disable_subscriptions {
```

Comment:
I would use positive or optimistic language instead of the opposite. `enable_subscriptions` where the default is true. That way we don't have to read `if not disable` but rather, `if enable`.

Reply:
The reason I went with disable is simply because I believe it is more common that the default value of a boolean configuration parameter is false, and it made more sense for subscriptions to be enabled by default. Hence I used `disable_subscriptions`. Wdyt @rakanalh  @eyusufatik ?

Reply:
done